src/test/rgw/rgw_multi/tests.py

```diff
@@ -962,9 +964,13 @@ def test_datalog_autotrim():
 
     # trim each datalog
     for zone, _ in zone_bucket:
+        started = datetime.datetime.now()
         datalog_autotrim(zone.zone)
         datalog = datalog_list(zone.zone)
-        assert len(datalog) == 0
+        # verify that all entries before trim are gone
+        for entry in datalog:
+            t = dateutil.parser.isoparse(entry['timestamp'])
+            assert(t >= started)
```

Comment:
maybe allow some margins (e.g .1 second)?
since the time is coming from 2 different systems?

Reply:
@cbodley ping

Reply:
thanks for review. i pushed an update that uses the markers instead of timestamps to avoid this timing business